README.md

```diff
@@ -34,6 +34,10 @@ when it's good to publish,
 2. Open the heroku dashboard at `https://dashboard.heroku.com/apps/salthouse-chat/deploy/github`
 3. Hit "Deploy Branch from Main"
 
+
```

Comment:
I like this. We'll need a separate branch to re-edit the READme, purpose statement, Analyze, Craft, Edit, etc.